azure-stack/hci/upgrade/install-enable-network-atc.md

```diff
@@ -227,27 +223,30 @@ ProvisioningStatus          : Completed
 
 Ensure that each intent added has an entry for the host you're working on. Also, make sure the **ConfigurationStatus** shows **Success**.
 
-If the **ConfigurationStatus** shows **Failed**, check to see if the error message indicates the reason for the failure. For some examples of failure resolutions, see [Common Error Messages](../deploy/network-atc.md#common-error-messages).
+If the **ConfigurationStatus** shows **Failed**, check to see if the error message indicates the reason for the failure. You can also review the Microsoft-Windows-Networking-NetworkATC/Admin event logs for more details on the reason for the failure. For some examples of failure resolutions, see [Common Error Messages](../deploy/network-atc.md#common-error-messages).
```

Comment:
@helmste - I will need to check with the PM on this. It looks like it will refer customers back to an older v22H2 article: https://learn.microsoft.com/en-us/azure-stack/hci/deploy/network-atc?tabs=22H2 and I want to make sure that aligns with the current direction.

Reply:
@ronmiab - the article already pointed to the 22H2 doc.  At this point, the customers are still on 22H2, so these docs would still be applicable.  The only thing I added was to look in the Network ATC event logs for more information on the failure.